geniza/urls.py

```diff
@@ -13,22 +13,51 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+
 from django.conf import settings
+from django.conf.urls import url
 from django.conf.urls.i18n import i18n_patterns
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import include, path
 from django.urls.conf import re_path
-from django.views.generic.base import RedirectView
+from django.views.generic.base import RedirectView, TemplateView
 from wagtail.admin import urls as wagtailadmin_urls
+from wagtail.contrib.sitemaps import Sitemap as WagtailSitemap
+from wagtail.contrib.sitemaps import views as sitemap_views
 from wagtail.core import urls as wagtail_urls
 
+from geniza.corpus.sitemaps import DocumentScholarshipSitemap, DocumentSitemap
+
+SITEMAPS = {
+    "pages": WagtailSitemap,
```

Comment:
When the content is first created, it's `None`, but after restarting the server, the links appear as expected. I added more content through the wagtail UI, and the sitemap updated properly without having to restart. I think it may have to do with the automatic generation of web pages.

This is an issue that wagtail hasn't addressed for 6 years, looking at this issue: https://github.com/wagtail/wagtail/issues/1196#issuecomment-890365995. In `geniza/venv/lib/python3.7/site-packages/wagtail/contrib/sitemaps/sitemap_generator.py` if I replace:
```diff
def location(self, obj):
- return obj.get_full_url(self.request)
+ return obj.localized.get_full_url(self.request)
```
...we wouldn't need to restart the server. Let me know what you think!

Reply:
is this resolved now ?

Reply:
Yes!